tests/playbooks/tests_802_1x.yml

```diff
@@ -3,6 +3,8 @@
 - hosts: all
   vars:
     interface: 802-1x-test
+  tags:
+    - tests::expfail
```

Comment:
In my opinion, it would be better to describe which platforms we want to skip as we do this for the wireless tests: https://github.com/linux-system-roles/network/blob/main/tests/ensure_provider_tests.py#L83

Also, if this is just because there is no EPEL 9, yet, then we also need to track this to ensure it will be enabled again at the right time.

Reply:
It has to do with the availability of the `hostapd` package, which comes from epel, and the downstream tests do not or cannot use epel